lib/MusicBrainz/Server/WebService/Serializer/JSON/LD/Release.pm

```diff
@@ -100,9 +100,10 @@ sub medium_format {
     # case of it not being one of these few formats. I'm not sure of the
     # best mitigation for either problem.
     my $name;
+    my $parent_format_id = $format->parent ? $format->parent->id : $format->parent_id;
```

Comment:
I also don't understand why `$format->parent_id` wouldn't always be identical to `$format->parent->id` here, but that's unrelated.